### PR TITLE
Settings, avatar and animation automation with XR engine bot 

### PR DESCRIPTION
**Pull request by SYBIOTE**
added bot features to simulate change in settings using e2e tests and integrated features into bot API
added bot features to simulate animations using e2e tests and integrated features into bot API
added bot feature to take screenshot on demand

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,12 +16,17 @@
   "scripts": {
     "check-errors": "tsc --noemit",
     "build-publish": "rimraf dist && cti entrypoint ./src -o index.ts && cross-env APP_ENV=production rollup -c",
-    "validate": "npm run test"
+    "validate": "npm run test",
+    "format": "prettier --write \"**/*.{ts,tsx}\""
```
Comment (HexaField) [inline package.json]: fantastic that you've added prettier, could you also add the pre-commit hooks & .prettierrc.json that the https://github.com/XRFoundation/XREngine repo has too? this way it is more standardized across all our projects

**Review by HexaField — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SYBIOTE) [reply package.json]: sure

Comment (SYBIOTE) [reply package.json]: done , updated to v1.1 as well

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "singleQuote": true,
+  "quoteProps": "as-needed",
+  "jsxSingleQuote": false,
+  "trailingComma": "none",
+  "bracketSpacing": true,
+  "bracketSameLine": false,
+  "arrowParens": "always",
+  "proseWrap": "never",
+  "endOfLine": "lf",
+  "importOrder": ["<THIRD_PARTY_MODULES>", "^@xrengine/(.*)$", "^@mui/(.*)$", "^[./]"],
```
Comment (HexaField) [inline .prettierrc.json]: need this to be `@etherealengine` instead of `@xrengine`

**Review by HexaField — CHANGES_REQUESTED** [review]

File: .prettierrc.json (hunk 2 above)
Comment (SYBIOTE) [reply .prettierrc.json]: on it

Comment (SYBIOTE) [reply .prettierrc.json]: done

**Review by HexaField — APPROVED** [review]

**Pull request merged**